src/lib/date-helper.js

```diff
@@ -84,13 +135,12 @@ function getTimeInTimeZone(dateString, timeZone = 0) {
   return `${newHours}:${newMinutes}:${newSeconds}`
 }
 
-const formatDate = (dateString) => {
-  const date = new Date(dateString);
-  const timestamp = date.getTime();
-
-  return `/Date(${timestamp})/`;
-};
+const formatDate = dateString => {
```

Comment:
is used formatDate ?? if not please clear

Reply:
i didn't add it; 
this needs to be checked in separate card